docs/root/configuration/observability/statistics.rst

```diff
@@ -24,7 +24,8 @@ Server related statistics are rooted at *server.* with following statistics:
   total_connections, Gauge, Total connections of both new and old Envoy processes
   version, Gauge, Integer represented version number based on SCM revision or :ref:stats_server_version_override` <envoy_api_field_config.bootstrap.v2.Bootstrap.header_prefix>` if set.
   days_until_first_cert_expiring, Gauge, Number of days until the next certificate being managed will expire
-  hot_restart_epoch, Gauge, Current hot restart epoch
+  hot_restart_epoch, Gauge, Current hot restart epoch -- an integer passed via command line flag ``--restart-epoch`, usually indicating generation.
```

Comment:
I went out on a limb here, staking a claim that this stat was just a pass-through from a command-line option, whereas hot_restart_generation auto-increments. Is that actually correct?

Reply:
This is actually making me wonder: could we get rid of the CLI option and/or just make it "--hot-restart" which would then get the generation, etc. from the parent?

Reply:
Yes I wasn't thinking in this PR. Can you potentially open a tracking issue?

Reply:
https://github.com/envoyproxy/envoy/issues/10139